globalConfig.json

```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:53470/2b5936fa-1c62-4b75-8b01-0181b614b5fe?"}
\ No newline at end of file
+{"mongoUri":"mongodb://127.0.0.1:53567/f567cabe-e253-4ee9-a278-ba2111812207?"}
```

Comment:
so we're fine including this?

Reply:
yeah idk tbh i guess it doesn't matter to push this or not...